integration/integration_tms_export_test.go

```diff
@@ -12,12 +12,13 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const testDockerImage = "alpine:latest"
```

Comment:
alpine latest works, it was most likely a missing certificate in the devxci image. I will open an issue in their repo to make them aware.